lms/djangoapps/courseware/utils.py

```diff
@@ -210,3 +212,19 @@ def get_course_hash_value(course_key):
         return int(m.hexdigest(), base=16) % 100
 
     return out_of_bound_value
+
+
+def _use_new_financial_assistance_flow(course_id):
+    """
+    Returns if the course_id can be used in the new financial assistance flow.
+    """
+    is_financial_assistance_enabled_for_course = WaffleFlagCourseOverrideModel.override_value(
+        ENABLE_NEW_FINANCIAL_ASSISTANCE_FLOW.name, course_id
+    )
+    financial_assistance_configuration = FinancialAssistanceConfiguration.current()
+    if financial_assistance_configuration.enabled and (
+            is_financial_assistance_enabled_for_course == WaffleFlagCourseOverrideModel.ALL_CHOICES.on or
```

Comment:
Wouldn't ENABLE_NEW_FINANCIAL_ASSISTANCE_FLOW.is_enabled(course_id) work here?

Reply:
No it doesn't seem to work like this. `is_enabled` doesn't take any argument.